README.md

```diff
@@ -183,6 +142,26 @@ Remember to write byte-saving code, see this [useful resource for JavaScript byt
 
 In every cool and modern browser.
 
+[ki.js](https://github.com/dciccale/ki.js/blob/master/ki.js) version is supported by all modern browsers: IE9+, Chrome 6+, Safari 5+, Firefox 6+, Opera 11.5+.
```

Comment:
I moved browser support a bit lower because it's not as important as at the time it was created. And also we support IE9+ which is already insane nowadays :D